code/obj/item/device/transfer_valve.dm

```diff
@@ -529,9 +551,10 @@ TYPEINFO(/obj/item/device/transfer_valve/briefcase)
 			boutput( user, "<b>There's no crystal in this here device!</b>")
 		else
 			if(crystal.pressure)
-				boutput( user, "The reader reads <b>[crystal.pressure/25]</b> kilojoules." )
+				boutput( user, "The reader reads <b>[crystal.pressure]</b> kiloblast." )
 			else
-				boutput( user, "The reader reads a firm 0. It guilts you into trying to read an unexploded pressure crystal, and seems to have succeeded. You feel ashamed for being so compelled by a device that has nothing more than a slot and a number display.")
+				boutput( user, "The reader reads a firm 0. It guilts you into trying to read an unexploded pressure crystal, and seems to have \
+					succeeded. You feel ashamed for being so compelled by a device that has nothing more than a slot and a number display.")
```

Comment:
I know these didn't have them before but they probably should have style classes.

Reply:
planning to atomize the bulk of the pressure sensor stuff into another pr